xtask/src/docgen.rs

```diff
@@ -48,6 +54,65 @@ pub fn typable_commands() -> Result<String, DynError> {
     Ok(md)
 }
 
+pub fn static_commands() -> Result<String, DynError> {
+    let mut md = String::new();
+    let keymap = EditorView::default().keymaps.map();
+    let keymaps = [
+        ("normal", keymap[&Mode::Normal].reverse_map()),
+        ("select", keymap[&Mode::Select].reverse_map()),
+        ("insert", keymap[&Mode::Insert].reverse_map()),
+    ];
+
+    md.push_str(&md_table_heading(&[
+        "Name".to_owned(),
+        "Description".to_owned(),
+        "Default keybinds".to_owned(),
+    ]));
+
+    for cmd in MappableCommand::STATIC_COMMAND_LIST {
+        let keymap_strings: Vec<_> = keymaps
+            .iter()
+            .map(|(mode, keymap)| {
+                let bindings = keymap
+                    .get(cmd.name())
+                    .map(|bindings| {
+                        let mut bind_strings: Vec<_> = bindings
+                            .iter()
+                            .map(|bind| {
+                                let keys = &bind
+                                    .iter()
+                                    .map(|key| key.key_sequence_format())
+                                    .collect::<String>();
+                                format!("`` {} ``", keys)
+                            })
+                            .collect();
+                        // sort for stable output. sort simple letters-only binds first, then lexicographic
+                        bind_strings.sort_by_key(|s| (s.contains('<'), s.to_owned()));
```

Comment:
Let me know if this is too cute. It makes sure the `h` is first here.

```md
| `move_char_left` | Move left | normal: `` h ``, `` <left> ``, insert: `` <left> `` |
| `move_char_right` | Move right | normal: `` l ``, `` <right> ``, insert: `` <right> `` |
```

Reply:
I think sorting by `s.len()` might make more sense? The byte length should be a more generic proxy for "shorter command sequence" than `.contains('<')` and might more effectively group similar binds like `page_up`